src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,138 @@
+            registrationService.register(userWithFiveSymbolsLogin);
+        });
+
+        assertThrows(NullPointerException.class, () -> {
+            registrationService.register(userWithNullLogin);
+        });
+
+        Storage.people.remove(userWithZeroSymbolsLogin);
+        Storage.people.remove(userWithFiveSymbolsLogin);
+        Storage.people.remove(userWithNullLogin);
+    }
+
+    @Test
+    void register_UserWithTooShortPassword_NotOk() {
+        User userWithZeroSymbolsPassword = new User(okLoginOne, "", okAge);
+        User userWithFiveSymbolsPassword = new User(okLoginTwo, "qwert", okAge);
+        User userWithNullPassword = new User(okLoginThree, null, okAge);
+
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(userWithZeroSymbolsPassword);
+        });
+
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(userWithFiveSymbolsPassword);
+        });
+
+        assertThrows(NullPointerException.class, () -> {
+            registrationService.register(userWithNullPassword);
+        });
+
+        Storage.people.remove(userWithZeroSymbolsPassword);
+        Storage.people.remove(userWithFiveSymbolsPassword);
+        Storage.people.remove(userWithNullPassword);
+    }
+
+    @Test
+    void register_UserWithTooYoungUser_NotOk() {
+        User userWithZeroAge = new User(okLoginOne, okPassword, 0);
+        User userWithNegativeAge = new User(okLoginTwo, okPassword, -100);
+        User userWithEdgeCaseBadAge = new User(okLoginThree, okPassword, 17);
+
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(userWithZeroAge);
+        });
+
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(userWithNegativeAge);
+        });
+
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(userWithEdgeCaseBadAge);
+        });
+
+        Storage.people.remove(userWithZeroAge);
+        Storage.people.remove(userWithNegativeAge);
+        Storage.people.remove(userWithEdgeCaseBadAge);
+    }
+
+    @Test
+    void register_UserWithRightParams_Ok() {
+        User userToRegisterOne = new User(okLoginOne, okPassword, okAge);
+        User userToRegisterTwo = new User(okLoginTwo, okPassword, okAge);
+        User userToRegisterThree = new User(okLoginThree, okPassword, okAge);
+
+        assertEquals(registrationService.register(userToRegisterOne), userToRegisterOne);
+        assertEquals(registrationService.register(userToRegisterTwo), userToRegisterTwo);
+        assertEquals(registrationService.register(userToRegisterThree), userToRegisterThree);
+
+        Storage.people.remove(userToRegisterOne);
+        Storage.people.remove(userToRegisterTwo);
+        Storage.people.remove(userToRegisterThree);
+    }
+
+    @Test
+    void register_UserWithRightButEdgeParams_Ok() {
```

Comment:
```suggestion
    void register_userWithRightButEdgeParams_ok() {
```
Use lowercase after underscore in method names